kili/services/asset_download/exceptions.py

```diff
@@ -0,0 +1,7 @@
+"""
+Asset download service exceptions
+"""
+
+
+class MissingPropertyError(Exception):
```

Comment:
You can inherit from [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) to be able to pass a message.